data/resume.yml

```diff
@@ -72,96 +76,93 @@ summary_links:
   - text: Technical Debt (Martin Fowler)
     link: http://martinfowler.com/bliki/TechnicalDebt.html
 brief_summary: |
-    Deeply experienced in software design and development methodologies,
-    especially related to the disciplines of software quality, web
-    applications, user experience, and team building. Presently working at the intersection of Rails, GraphQL, and React, developing modern web applications with fast response times.
+    Deeply experienced in software architecture, design, and development methodologies,
+    especially related to the disciplines of web applications, user experience, team
+    building, and processes. Working at the intersection of Rails, GraphQL, and React,
+    developing modern web applications with fast response times.
 skills:
   general:
-    - software engineering
-    - web development
     - business analysis
     - consulting
     - facilitating
+    - software engineering
     - teaching
+    - web development
   software_engineering:
+    - "APIs, REST, GraphQL"
```

Comment:
This was good to move up to the top, but how you had it before was clearer.
suggestion:
```
APIs: REST, GraphQL
```

Reply:
@thekbb : I wonder how to express both using and creating APIs in different forms...